XenModel/XenModel.csproj

```diff
@@ -88,7 +88,7 @@
     <Compile Include="Actions\Host\HostAbstractAction.cs" />
     <Compile Include="Actions\Host\RebootHostAction.cs" />
     <Compile Include="Actions\Host\RestartToolstackAction.cs" />
-    <Compile Include="Actions\Host\NRPEUpdateAction.cs" />
+    <Compile Include="Actions\NRPE\NRPEUpdateAction.cs" />

```

Comment:
I moved these two files together in the same folder. We'll use this folder to also add a new action that will be fetching the configuration as per this comment https://github.com/xenserver/xenadmin/pull/3226#discussion_r1340123271